example/objects.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>flowchart.js · Playground</title>
+    <style type="text/css">
+        .end-element {
+            background-color: #FFCCFF;
+        }
+    </style>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/raphael/2.1.0/raphael-min.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <!--<script src="http://flowchart.js.org/flowchart-latest.js"></script>-->
+    <script src="../release/flowchart.js"></script>
+    <script>
+
+        window.onload = function () {
+
+            var chart;
+                    if (chart) {
+                chart.clean();
+            }
+
+
+            var data = {
+                symbols: {
+                    st: {
+                        key: 'st',
```

Comment:
Seems a bit like a maintenance nightmare to have a property in all these objects with the key name. You can always get the key name with builtin functionality.